### PR TITLE
add extractors in new format

### PR DESCRIPTION
**Pull request by rishiraj**

### CODE REVIEW

**Comment by rishiraj** [comment]
@diptanu added table and image extraction complete code.

File: docs/docs/usecases/pdf_extraction.md [hunk 1]
```diff
@@ -24,38 +24,16 @@ If all you want is to extract text, images, and tables from a PDF, this is the o
 ## Image Extraction
 If you would like to extract images from PDF, the best extractor to use is `tensorlake/pdfextractor` It automatically extracts images from documents and writes them into blob stores. 
 
-You can get extracted images from pdfextractor with a simple python code like this:
```
Comment (diptanu) [inline docs/docs/usecases/pdf_extraction.md:27]: You can keep the example here, and link to the docs.

File: docs/docs/examples/multimodal_translation/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
+# PDF Translation with Indexify and GPT-4o
```
Comment (diptanu) [inline docs/docs/examples/multimodal_translation/README.md:1]: Can you move them into a PDF folder?

Comment (diptanu) [reply docs/docs/examples/multimodal_translation/README.md:1]: Rename the folder to gpt4o_pdf_translation

File: docs/docs/examples/regular_translation/pdf_translation_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from indexify import IndexifyClient, ExtractionGraph
+
+client = IndexifyClient()
+
+extraction_graph_spec = """
```
Comment (diptanu) [inline docs/docs/examples/regular_translation/pdf_translation_pipeline.py:5]: Rename the folder to gpt4_translation

Comment (diptanu) [reply docs/docs/examples/regular_translation/pdf_translation_pipeline.py:5]: Both of these cookbooks are very much the same, other than the model. Can we merge them and rename the folder to openai_pdf_translation and show the multimodal and regular translation in two sections, while rest of the code is same?

**Review by diptanu — CHANGES_REQUESTED** [review]

File: docs/docs/usecases/pdf_extraction.md (hunk 1 above)
Comment (rishiraj) [reply docs/docs/usecases/pdf_extraction.md:27]: that example was obsolete. it was from a time when we didn’t have the new uniform format of pdf extractors. now we can simply put [“tables”] in the graph itself and that would do. do you want me to add the graph here?

File: docs/docs/examples/regular_translation/pdf_translation_pipeline.py (hunk 3 above)
Comment (rishiraj) [reply docs/docs/examples/regular_translation/pdf_translation_pipeline.py:5]: ok doing this.

File: docs/docs/usecases/pdf_extraction.md (hunk 1 above)
Comment (diptanu) [reply docs/docs/usecases/pdf_extraction.md:27]: Yeah lets add the graph

**Review by diptanu — CHANGES_REQUESTED** [review]

**Pull request merged**